wgengine/monitor/monitor_windows.go

```diff
@@ -100,23 +107,40 @@ func (m *winMon) Receive() (message, error) {
 
 	select {
 	case msg := <-m.messagec:
-		m.logf("got windows change event after %v: evt=%s", time.Since(t0).Round(time.Millisecond), msg.eventType)
+		now := time.Now()
+		m.mu.Lock()
+		sinceLast := now.Sub(m.lastLog)
+		m.lastLog = now
+		m.mu.Unlock()
```

Comment:
should we only update `m.lastLog` if it is a `ts` event? either way add a comment to why

Reply:
Commented.